main.rb

```diff
@@ -1,41 +1,44 @@
-require_relative 'person'
-require_relative 'nameable'
-require_relative 'decorator'
-require_relative 'trimmer_decorator'
-require_relative 'capitalize_decorator'
-require_relative 'classroom'
-require_relative 'student'
-require_relative 'book'
-require_relative 'rental'
+require_relative 'app'
 
-person = Person.new(22, 'maximilianus')
-puts "Person's Correct Name: #{person.correct_name}"
-capitalized_person = CapitalizeDecorator.new(person)
-puts "Capitalized Person's Correct Name: #{capitalized_person.correct_name}"
-capitalized_trimmed_person = TrimmerDecorator.new(capitalized_person)
-puts "Capitalized and Trimmed Person's Correct Name: #{capitalized_trimmed_person.correct_name}"
-classroom = Classroom.new('Math Class')
-# Create students
-student1 = Student.new(classroom, 18, 'Alice')
-student2 = Student.new(classroom, 22, 'das')
-student3 = Student.new(classroom, 33, 'acs')
-puts "Classroom Label: #{classroom.label}"
-puts "Classroom Students: #{classroom.students.map(&:name).join(', ')}"
-puts "Student1 Classroom: #{student1.classroom&.label}"
-puts "Student2 Classroom: #{student2.classroom&.label}"
-puts "Student3 Classroom: #{student3.classroom&.label}"
-book1 = Book.new('The Great Gatsby', 'F. Scott Fitzgerald')
-person1 = Person.new(22, 'Maximilianus')
-rental1 = Rental.new('2023-08-03', book1, person1)
-puts rental1.person
-# Add the rental to the book's rentals and person's rentals
-book1.add_rental(rental1, person1)
-person1.add_rental(rental1, book1)
-puts "Rentals for #{book1.title}:"
-book1.rentals.each do |rental|
-  puts "#{rental.date}, Rented by: #{rental.person.name}"
+def render_choices
+  puts 'Choose your Option:'
+  puts '1. List all books.'
+  puts '2. List all people.'
+  puts '3. Create a person.'
+  puts '4. Create a book.'
+  puts '5. Create a rental.'
+  puts '6. List all rentals for a given person id.'
+  puts '7. Quit'
 end
-puts "Rentals for #{person1.name}:"
-person1.rentals.each do |rental|
-  puts "#{rental.date}, Book: #{rental.book.title}"
+
+def choose_a_number(choice, app)
+  case choice
+  when 1
+    app.list_books
+  when 2
+    app.list_people
+  when 3
+    app.create_a_person
+  when 4
+    app.create_a_book
+  when 5
+    app.create_a_rental
+  when 6
+    app.list_rentals
+  else
+    puts '👋 Sorry to see you leaving..'
+  end
 end
+
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you added some more meaningful features in the Future Feature section because right now you are adding a UI so it would be nice if you add other features in your Future Feature section which you will add in the future 👍 to make your readme descriptive 😊
- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍